IBAnimatable/AnimatableBarButtonItem.swift

```diff
@@ -45,6 +45,6 @@ open class AnimatableBarButtonItem: UIBarButtonItem, BarButtonItemDesignable, An
   // MARK: - Private
   fileprivate func configureInspectableProperties() {
 //    configureAnimatableProperties()
-    confingBarButtonItemImage()
+    configBarButtonItemImage()
```

Comment:
Can you change it to `configureBarButtonItemImage`, we changed all `config` to `configure` when we upgraded to Swift 3. this one was missed. ☹️